tests/core/daemon/test_daemon.py

```diff
@@ -617,12 +617,16 @@ def get_keys_no_secrets(self, include_secrets):
             "success": True,
             "keys": {
                 test_key_data.fingerprint: {
-                    "farmer_public_key": "b800fa5c8fe8890bf9b8c8ad7e981aa3face3ae2296f7df7673e9b63ddeccbac9723cfe9968b7530179cd7183d261b3e",
-                    "pool_public_key": "b1b0d8e777e3b1e79f33bc8707c74c8e4907fd8c3da870d781803f647b17f3cc6250f09362f25699eb0ee385c57a2e01",
+                    "farmer_public_key": "b800fa5c8fe8890bf9b8c8ad7e981aa3face3ae2296f7df7"
+                    "673e9b63ddeccbac9723cfe9968b7530179cd7183d261b3e",
+                    "pool_public_key": "b1b0d8e777e3b1e79f33bc8707c74c8e4907fd8c3da870d7"
+                    "81803f647b17f3cc6250f09362f25699eb0ee385c57a2e01",
                 },
                 test_key_data_2.fingerprint: {
-                    "farmer_public_key": "a96a194fe76043b4867fa154b1fcda6709257699a720cc10af68a8414d9bf08cee0945becf9b36a1c76d5b04d379bc7f",
-                    "pool_public_key": "837646f1378b1194aa1054b8869dcea5006cb173730ceeb90e9b9bb7f5d73e481775d20a1726fec3ef140c5b9cdbd4b3",
+                    "farmer_public_key": "a96a194fe76043b4867fa154b1fcda6709257699a720cc10"
+                    "af68a8414d9bf08cee0945becf9b36a1c76d5b04d379bc7f",
+                    "pool_public_key": "837646f1378b1194aa1054b8869dcea5006cb173730ceeb9"
+                    "0e9b9bb7f5d73e481775d20a1726fec3ef140c5b9cdbd4b3",
```

Comment:
it's not easy to tell that these numbers make sense and that the test is correct. can you add some comments (or compute these in some way that makes it clear), to explain to the reader of what these test cases exercise any where these numbers come from?

Reply:
I changed the code to derive keys from the original test private keys.

Reply:
Resoving comment per code change